filter_temporal.json

```diff
@@ -73,7 +73,7 @@
         }
     ],
     "returns": {
-        "description": "A data cube restricted to the specified temporal extent. The dimensions and dimension properties (name, type, labels, reference system and resolution) remain unchanged, except that the given temporal dimension(s) have less (or the same) dimension labels.",
+        "description": "A data cube restricted to the specified temporal extent. The dimensions and dimension properties (name, type, labels, reference system and resolution) remain unchanged, except the temporal dimensions may have less dimension labels depending on the value for the `dimension` parameter.",
```

Comment:
```
... except the temporal dimensions may have less dimension labels depending on the value for the `dimension` parameter.",
```

Everything else looks good, I would just rephrase this sentence, because it could be understood as if the labels depend on `dimension` parameter (they depend on `extent`).

I would suggest:
```
...except the temporal dimensions (determined by `dimensions` parameter) may have less dimension labels.",
```

Not a big deal though, it is clear from context what is meant.

Reply:
Good suggestion!

```suggestion
        "description": "A data cube restricted to the specified temporal extent. The dimensions and dimension properties (name, type, labels, reference system and resolution) remain unchanged, except that the temporal dimensions (determined by `dimensions` parameter) may have less dimension labels.",
```